src/sweets/core.py

```diff
@@ -75,6 +75,20 @@ class Workflow(YamlModel):
             " by aria2)."
         ),
     )
+    dem_filename: Path = Field(
+        # requires that `work_dir` is specified earlier than `dem_filename`
+        default_factory=lambda data: data["work_dir"] / "dem.dat",
```

Comment:
```suggestion
        default_factory=lambda data: data["work_dir"] / "dem.tif",
```
Is `.dat` your extension for the flat binary version?

Reply:
Yes, but I like a `.tif` file better